dcos/dcos-ui
# Resizable Nodes and Services table columns

## Testing
(recommended in a private browsing tab)
1. Navigate to the {Nodes|Services} page
2. Resize some columns
3. Refresh the page

Columns should be resizable, and when you refresh the page, the resized columns should be the same width you resized them to

## Trade-offs



## Dependencies



## Screenshots
![ServicesResizeDemo](https://user-images.githubusercontent.com/2313998/55919451-11b06000-5bc4-11e9-94bd-1dcb9fecb9ac.gif)
